integration-tests/emulated/chains/relays/kusama/src/genesis.rs

```diff
@@ -18,7 +18,6 @@ use authority_discovery_primitives::AuthorityId as AuthorityDiscoveryId;
 use babe_primitives::AuthorityId as BabeId;
 use beefy_primitives::ecdsa_crypto::AuthorityId as BeefyId;
 use grandpa::AuthorityId as GrandpaId;
-use pallet_im_online::sr25519::AuthorityId as ImOnlineId;
```

Comment:
please also remove [`pallet-im-online` dep from Cargo.toml](https://github.com/polkadot-fellows/runtimes/blob/59172d5fb8feea2646490618bead3b2b4fb06220/integration-tests/emulated/chains/relays/kusama/Cargo.toml#L20)

Reply:
Thanks for the catch! Removed here and from chainspec generator as well.